app/models/version.rb

```diff
@@ -171,6 +171,10 @@ def platformed?
 
   delegate :reorder_versions, to: :rubygem
 
+  def can_yank
```

Comment:
What do you think about adding a `?` to this method? So it becomes `can_yank?`? Or maybe `yankable?`?